pkg/dunner/dunner.go

```diff
@@ -51,6 +51,10 @@ func Do(_ *cobra.Command, args []string) {
 func ExecTask(configs *config.Configs, taskName string, args []string) {
 	var async = viper.GetBool("Async")
 	var wg sync.WaitGroup
+
+	if _, exists := configs.Tasks[taskName]; !exists {
+		log.Errorf("dunner: task '%s' does not exist", taskName)
```

Comment:
This should become a part of validation and not during dunner execution.

Reply:
`taskName` is given by the CLI and not the dunner file. I believe validation is done for the dunner file. Could you please explain with file and line number, where should the check be implemented?